cloud/pkg/admissioncontroller/admission.go

```diff
@@ -158,6 +165,7 @@ func (ac *AdmissionController) registerWebhooks(opt *options.AdmissionOptions, c
 						Namespace: opt.AdmissionServiceNamespace,
 						Name:      opt.AdmissionServiceName,
 						Path:      strPtr("/devicemodels"),
+						Port:      int32Ptr(opt.Port),
```

Comment:
I think we can change port type to `int32` in opts? @chendave

Reply:
thank you for the comments, there is no int32 flags as IntVar. we cannot make it as a int32 options. 

see: https://github.com/kubeedge/kubeedge/blob/master/cloud/cmd/admission/app/options/options.go#L50

Reply:
How about use `Int32Var` instead?  https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/options/replicasetcontroller.go#L36